Hi3Helper.Core/Lang/en.json

```diff
@@ -394,67 +394,67 @@
     "ChooseAudioLangSelectPlaceholder": "Select Your Audio Language",
     "ChooseAudioLangTitle": "Choose Audio Language",
-    "ChooseAudioLangSubtitle": "Before you install the game, you need to choose which audio language you want to use (Default: Japanese):",
+    "ChooseAudioLangSubtitle": "Before installing the game, you need to choose which audio language you want to use (Default: Japanese):",
     "AddtDownloadNeededTitle": "Additional Download is Needed",
-    "AddtDownloadNeededSubtitle": "You have to download atleast {0} of additional/persistent files. Although you can skip and download it in-game instead.\nDo you want to continue?",
-    "AddtDownloadCompletedTitle": "Additional Download is Completed!",
-    "AddtDownloadCompletedSubtitle": "Additonal/Persistent Download has been completed! You can now enjoy your game.\nHappy gaming!",
-    "RepairCompletedTitle": "Repair Process is Completed!",
+    "AddtDownloadNeededSubtitle": "You must download at least {0} of additional/persistent files. You can also skip this process and download it in-game instead.\nWould you like to download it now?",
+    "AddtDownloadCompletedTitle": "Additional Download Completed!",
+    "AddtDownloadCompletedSubtitle": "Additonal/Persistent Download has been completed! \nHappy gaming!",
+    "RepairCompletedTitle": "Repair Process Completed!",
     "RepairCompletedSubtitle": "{0} file(s) have been repaired.",
-    "RepairCompletedSubtitleNoBroken": "No files are broken.",
-    "ExtremeGraphicsSettingsWarnTitle": "Very High Setting Selected!",
-    "ExtremeGraphicsSettingsWarnSubtitle": "You are about to set the setting to Very High!\r\nRemember that Very High setting is basically 2x Render Scale with MSAA Enabled and it's VERY UNOPTIMIZED! (unless you have a NASA Super Computer to run it).\r\n\r\nAre you sure to use this setting?",
-    "ExistingInstallTitle": "Existing Installation is Detected!",
-    "ExistingInstallSubtitle": "Game is already been installed on this location:\r\n\r\n{0}\r\n\r\nIt's recommended to migrate the game to Collapse Launcher.\r\nHowever, you can still use Official Launcher to start the game.\r\n\r\nDo you want to continue?",
-    "ExistingInstallBHI3LTitle": "Existing Installation on BetterHI3Launcher is Detected!",
-    "ExistingInstallBHI3LSubtitle": "Game is already been installed on this location:\r\n\r\n{0}\r\n\r\nIt's recommended to migrate the game to Collapse Launcher.\r\nHowever, you can still use BetterHi3Launcher to start the game.\r\n\r\nDo you want to continue?",
+    "RepairCompletedSubtitleNoBroken": "No broken files found.",
+    "ExtremeGraphicsSettingsWarnTitle": "Very High Setting Preset Selected!",
+    "ExtremeGraphicsSettingsWarnSubtitle": "You are about to set the setting preset to Very High!\r\The Very High setting preset is basically 2x Render Scale with MSAA Enabled and it's VERY UNOPTIMIZED! (unless you have a NASA Supercomputer to run it).\r\n\r\nAre you sure you want to use this setting?",
```

Comment:
Missing `n` in ```... preset to Very High!\r\(n)The Very High ...``` to make new line
 I think you can also remove `The` as it feels unnecessary.

Reply:
Fair point, I'll make the corrections for this.